libs/sdk-backend-spi/src/workspace/fromModel/ldm/metadata/measure/index.ts

```diff
@@ -18,6 +18,31 @@ export interface IMeasureMetadataObject extends IMetadataObject {
      * Measure formatting
      */
     format: string;
+
+    /**
+     * Measure is locked for editing
+     */
+    isLocked?: boolean;
+}
+
+/**
+ * Measure metadata object definition
+ *
+ * @public
+ */
+export interface IMeasureMetadataObjectDefinition
```

Comment:
We ususally have these set up so the object extends its definition (for example IInsight and IInsightDefinition). Can it be done here as well?